### PR TITLE
Fix/#161

### PR DESCRIPTION
**Pull request by e-azocar**
### Description

Edited and fixed some files that have `// @ts-nocheck` or `//@ ts-ignore`.

And I passed BackButton text as a children

fixes # 161

#### Screenshots

image
image

### CODE REVIEW

**Comment by Israel-Laguan** [comment]
These changes are present in #162 so will only check that PR

File: src/pages/Confirm/index.jsx [hunk 1]
```diff
@@ -42,13 +51,20 @@ const Confirm = () => {
...
-      confirmPackage(event, packageId)
+    // @ts-ignore
+    if (event.RUT) {
+      // @ts-ignore
+
+      confirmPackage(event, id)
     } else {
       alert('Por favor complete los campos')
     }
   }
```
Comment (heliomar-pena) [inline src/pages/Confirm/index.jsx:62]: Why here is changed the condition?

File: src/pages/Confirm/index.jsx [hunk 2]
```diff
@@ -87,9 +112,7 @@ const Confirm = () => {
         <Form onSubmit={e => submitConfirmation(e)}>
           {({ handleFormChange, value }) => (
             <>
-              <FormInput placeholder="Nombre" name="name" value={value} onChange={handleFormChange} />
               <FormInput placeholder="RUT" name="RUT" value={value} onChange={handleFormChange} />
-              <FormInput placeholder="Celular" name="phone" value={value} onChange={handleFormChange} />
```
Comment (heliomar-pena) [inline src/pages/Confirm/index.jsx:92]: Why here is only using the RUT? @Israel-Laguan is a change of the customer?

**Review by heliomar-pena — COMMENTED** [review]
The check page is not redirecting me to map page :( Only show the modal but when I click in confirmar is not redirecting me

imagen

File: src/pages/Confirm/index.jsx (hunk 1 above)
Comment (e-azocar) [reply src/pages/Confirm/index.jsx:62]: Yes, here we are using only the RUT to confirm the package

File: src/pages/Confirm/index.jsx (hunk 2 above)
Comment (e-azocar) [reply src/pages/Confirm/index.jsx:92]: Here's happening same. @Israel-Laguan and me chatted about this and did it this

**Pull request closed without merging**